.gitignore

```diff
@@ -5,3 +5,5 @@ share/scripts/OSOM/*
 .DS_Store
 Executable/
 share/datasets/
+external/
```

Comment:
if external only has git submodules, github will just show them as gray folders and not actually sync all the contents

Reply:
I removed git submodules. should we still keep this folder?